api/app/src/Service/OrgService.php

```diff
@@ -47,13 +41,11 @@ class OrgService
     public function __construct(
         EntityManagerInterface $em,
         UserRepository $userRepository,
-        ClientRepository $clientRepository,
```

Comment:
You've gone ahead & used constructor promotion elsewhere in this PR, small little refactor on this section as well?